### PR TITLE
Improve explanations of `custom-settings` in the README

### LINKED ISSUE #11: Use facets
Make the scraper use the facets (when the facets feature will be ready in a stable version of MeiliSearch)

### PR DESCRIPTION
**Pull request by Smartmind12**
# Pull Request

## Related issue
Fixes #11 

## What does this PR do?
- Added filterableAttributes as a Setting option in read.me file.

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

**Review by alallema — COMMENTED** [review]
Thanks, @Smartmind12 for your PR!
Indeed the issue said to add the `filterable-attribute` in the README but I think that it weighs down the readme and I think that the best would be to remove all the three paragraphs about `synonyms`, `stopWords ` and `filterable-attributes` add only the line `filterable-attribute` in the example already present with a title like and a link a the end:

Example:
```json
"custom_settings": {
    "synonyms": {
      "static site generator": [
        "ssg"
      ],
      "ssg": [
        "static site generator"
      ]
    },
    "stopWords": ["of", "the"],
    "filterableAttributes": ["genres", "type"],
  },
```
Learn more about `filterableAttributes`, `synonyms` and `stop-words` and all the other settings in [the Meilisearch documentation](https://docs.meilisearch.com/reference/api/settings.html#settings).

Hope it's clear enough don't hesitate if you had some questions

**Comment by Smartmind12** [comment]
> the Meilisearch documentation

Yup that makes sense! I have updated the same in the commit.

File: README.md [hunk 1]
```diff
@@ -424,32 +424,13 @@ This field can be used to add Meilisearch settings.
...
-
-```json
-"custom_settings": {
+    "stopWords": ["of", "the"],
     "filterableAttributes": ["genres", "type"],
-    "stopWords": ["of", "the"]
   },
 ```
 
+Learn more about `filterableAttributes`, `synonyms` and `stop-words` and all the other settings in the [Meilisearch documentation](https://docs.meilisearch.com/reference/api/settings.html#settings-object).
```
Comment (alallema) [inline README.md:432]:
```suggestion
Learn more about `filterableAttributes`, `synonyms`, `stop-words` and all available settings in the [Meilisearch documentation](https://docs.meilisearch.com/reference/api/settings.html#settings-object).
```

**Review by alallema — COMMENTED** [review]
Hi @Smartmind12,
Thank you. That's much better, just adding a suggestion. Also can you add:
`Example:`
before the settings example on the README?

**Comment by Smartmind12** [comment]
> Hi @Smartmind12, Thank you. That's much better, just adding a suggestion. Also can you add: `Example:` before the settings example on the README?

Yupp its done!

**Review by alallema — APPROVED** [review]
Thanks @Smartmind12 for this PR!
LGTM! 🎉

File: README.md [hunk 2]
```diff
@@ -46,6 +46,7 @@ This scraper is used in production and runs on the [Meilisearch documentation](h
     - [`sitemap_alternate_links` (optional)](#sitemap_alternate_links-optional)
     - [`selectors_exclude` (optional)](#selectors_exclude-optional)
     - [`custom_settings` (optional)](#custom_settings-optional)
+    - [`filterableAttributes` (optional)](#filterableAttributes-optional)
```
Comment (alallema) [inline README.md:49]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -423,20 +424,12 @@ This field can be used to add Meilisearch settings.
         "static site generator"
       ]
     },
-    "stopWords": ["of", "the"]
+    "stopWords": ["of", "the"],
+    "filterableAttributes": ["genres", "type"],
   },
```
Comment (alallema) [inline README.md:429]:
```suggestion
    "filterableAttributes": ["genres", "type"],
  }
```

File: README.md [hunk 4]
```diff
@@ -423,20 +424,12 @@ This field can be used to add Meilisearch settings.
         "static site generator"
       ]
     },
-    "stopWords": ["of", "the"]
-  },
+    "stopWords": ["of", "the"],
+    "filterableAttributes": ["genres", "type"],
```
Comment (alallema) [inline README.md:428]:
```suggestion
    "filterableAttributes": ["genres", "type"]
```

**Comment by alallema** [comment]
bors merge

**Review by alallema — APPROVED** [review]

**Comment by alallema** [comment]
_This message is sent automatically_

Thanks again for contributing to Meilisearch :heart:
If you are participating in [Hacktoberfest](https://blog.meilisearch.com/contribute-hacktoberfest-2022/), and you would like to receive some gift from Meilisearch too, please complete [this form](https://meilisearch.typeform.com/to/idt5qugY).

**Pull request merged**